extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -243,15 +243,16 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
     const result: ParsedRecipients = { to: [], cc: [], bcc: [] };
     for (const recipient of recipients) {
       /* eslint-disable @typescript-eslint/no-non-null-assertion */
+      const recipientObj = { email: recipient.email, name: Xss.stripEmojis(recipient.name || '') };
```

Comment:
It seems here we remove emojis even for recipients who have encryption (so we won't use FES for sending messages), and emoji in recipient name should work for them.

Probably it'll be enough to remove emojis just on creating password-protected message before sending it to FES.

Reply:
Upon retesting, the emoji needs to be stripped even in this phase otherwise the error will occur when the email is about to be sent and reaches the FES server. I'll try to look up on other solution.

Reply:
I think emojis should be stripped when formatting password messages in [formatSendablePwdMsgs](https://github.com/FlowCrypt/flowcrypt-browser/blob/ab31c96d09b3a2f86f82cfbbbd55fbd5c28cc172/extension/chrome/elements/compose-modules/formatters/encrypted-mail-msg-formatter.ts#L94)

Reply:
Ah yes, that is right - I have updated the code and manipulated the `newMsg` object inside the formatSendablePwdMsgs function. Thank you!